gis/tippecanoe/Portfile

```diff
@@ -6,17 +6,17 @@ PortGroup               makefile 1.0
 PortGroup               legacysupport 1.0
 PortGroup               compiler_blacklist_versions 1.0
 
-github.setup            felt tippecanoe 2.45.0
+github.setup            felt tippecanoe 2.49.0
 revision                0
```

Comment:
please add "github.tarball_from" line and set it to "releases" or "archive" - depending on what upstream provides. It will require updating the checksums.

Reply:
Added.